src/scripts/browserDeviceProfile.js

```diff
@@ -692,6 +692,8 @@ define(['browser'], function (browser) {
         var h264Profiles = 'high|main|baseline|constrained baseline';
 
         if (browser.tizen || browser.web0s ||
+            // iOS 12 incorrectly reports support
+            !(browser.iOS && browser.iOSVersion < 13) &&
```

Comment:
Making a function like `canPlayH264Level51` or something would simplify the complexity of this a bit.

Reply:
:+1: updated.